modules/signing_worker/files/update-requirements.sh

```diff
@@ -15,17 +16,39 @@ popd &>/dev/null || exit
 # be different
 cwd=$(pwd)
 
+if [ -z "$1" ]; then
+    echo "Updating all requirement files"
+    for f in ${MY_DIR}/requirements.*; do
+        target=$(echo $f | awk -F '.' '{print $2}')
+        echo "Updating ${f}"
+        ${MY_DIR}/${0} $target $f
```

Comment:
Nit: Let's quote args in order to avoid shell injection.

```suggestion
        "${MY_DIR}/${0}" "$target" "$f"
```